src/components/decoration/sprite/index.tsx

```diff
@@ -49,19 +48,31 @@ const Sprite: React.FC<SpriteProps> = ({
 
   const initialPositionPx = isMdScreen ? 0 : pixelsLeft * pixelSize;
   const finalPositionPx = isMdScreen
-    ? width - image.width * scale
+    ? width - image.width * scale - 20
```

Comment:
I think we should just hide the animals on <md resolutions, as this is how our other pages handle it.
Also, isn't this the final position for md screens, not mobile?

Reply:
I'm not sure I follow, this is the component we use for all animated animals, so we're not hiding them on mobile? :thinking: 

Screens < md and mobile are almost always the same thing, and in most cases md is a good point to change the layout (sm might be too small)

Reply:
Sorry, I confused the hiding of blocks with the sprites.

Right, I assumed `isMdScreen` was >md :D